src/main/java/net/sf/jabref/gui/desktop/JabRefDesktop.java

```diff
@@ -432,4 +434,37 @@ public static void openBrowser(String url) throws IOException {
         ExternalFileType fileType = ExternalFileTypes.getInstance().getExternalFileTypeByExt("html");
         openExternalFilePlatformIndependent(fileType, url);
     }
+
+    public static void openConsole(File file) throws IOException {
```

Comment:
please add if(file == null) { return;} so that no exception occurs if this method is called wrongly.